extensions/binding/org.eclipse.smarthome.binding.onewire/src/main/java/org/eclipse/smarthome/binding/onewire/internal/discovery/OwDiscoveryItem.java

```diff
@@ -151,16 +126,16 @@ public String getVendor() {
      *
      * @return
      */
```

Comment:
Please complete the JavaDoc (@return annotation), also review the other methods.

Reply:
Yes. But I fixed the doc in another file where it was missing for the same method.

Reply:
Are you going to complete it here, too? it looks half finished like this.

Reply:
Sure.